cpp/models/ode_secir/parameters.h

```diff
@@ -360,19 +360,23 @@ class Parameters : public ParametersBase
 
     /**
      * @brief checks whether all Parameters satisfy their corresponding constraints and applies them, if they do not
+     * @return Returns 1 if one ore more constraint was corrected, 0 otherwise.  
      */
-    void apply_constraints()
+    int apply_constraints()
```

Comment:
Not a must but just a question: Does a `boolean` not make more sense if we do not use other values than 0 and 1?